Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -73,17 +74,20 @@ class MainMenuActionHelper: PhotonActionSheetProtocol,
     ///   - profile: the user's profile
     ///   - tabManager: the tab manager
     ///   - buttonView: the view from which the menu will be shown
+    ///   - toastContainer: the view hosting a toast alert
     ///   - showFXASyncAction: the closure that will be executed for the sync action in the library section
     init(profile: Profile,
          tabManager: TabManager,
          buttonView: UIButton,
+         toastContainer: UIView? = nil,
```

Comment:
In which case is this `toastContainer` nil? I think it's always the `alertContainer` ?